apps/condo/domains/property/schema/Property.js

```diff
@@ -146,6 +149,13 @@ const Property = new GQLListSchema('Property', {
                 return data.inwork.count
             },
         },
+
+        rankIndicator: {
+            schemaDoc: 'Property rankIndicator, used for rankedProperties query, has a 1:1 ratio with total tickets which associated with single property',
+            type: Integer,
+            required: false,
+            defaultValue: 0,
```

Comment:
please add an `access`. Like `User.isPhoneVerified`

Reply:
Field is depricated